configure.ac

```diff
@@ -15,6 +15,8 @@ AM_SILENT_RULES([yes])
 AC_GNU_SOURCE
 AC_PROG_CC
 
+AC_SYS_LARGEFILE
```

Comment:
I think we should also fix printing `off_t` values, I found one in `pread()` error reporting in proc.c:copy_memory_proc_mem()

Reply:
Done: https://github.com/tbricks/tbstack/pull/21/commits/052355b107df88dc43ab484aac0387a87e39f327